sqlitecluster/SQLiteNode.cpp

```diff
@@ -776,12 +921,13 @@ bool SQLiteNode::update() {
             if (numFullDenied || (everybodyResponded && !consistentEnough)) {
                 SINFO("Rolling back transaction because everybody currently connected responded "
                       "but not consistent enough. Num denied: " << numFullDenied << ". Follower write failure?");
-                _db.rollback();
 
                 // Notify everybody to rollback
                 SData rollback("ROLLBACK_TRANSACTION");
                 rollback.set("ID", _lastSentTransactionID + 1);
+                rollback.set("NewHash", _db.getUncommittedHash());
```

Comment:
Just confirming, the follower doesn't actually need this, right? It's harmless to send, however.

Reply:
We needed this in an earlier revision that kept hashes of transactions to rollback but no longer need it since we've removed that. I should be able to remove this here.